SDL/LST.cc

```diff
@@ -10,8 +10,7 @@ void SDL::LST::eventSetup() {
     SDL::initModules(path);
 }
 
-void SDL::LST::run(cudaStream_t stream,
```

Comment:
I am afraid this won't work for CMSSW. We need to pass the queue managed by the framework, so that the kernels run there. I guess one solution would be to have another constructor which is accepting and using an externally provided queue (I haven't tested this and the tests should ideally go along with the fix)?

Reply:
I see. Yeah that sounds reasonable to have a second constructor.